miio/integrations/vacuum/mijia/g1vacuum.py

```diff
@@ -373,3 +374,22 @@ def consumable_reset(self, consumable: G1Consumable):
     def set_fan_speed(self, fan_speed: G1FanSpeed):
         """Set fan speed."""
         return self.set_property("fan_speed", fan_speed.value)
+
+    @command()
+    def fan_speed_presets(self) -> Dict[str, int]:
+        """Retrieve predefined fan speeds, dictionary where.
+
+        - key is name (identifier)
+        - value is integer representation usable as argument for set_fan_speed_preset() method
+        """
+        return {x.name: x.value for x in G1FanSpeed}
+
+    @command(click.argument("speed", type=int))
+    def set_fan_speed_preset(self, speed: int) -> None:
+        """Sets fan speed preset value.
```

Comment:
```suggestion
    def set_fan_speed_preset(self, preset: int) -> None:
        """Set fan speed preset.
```
Maybe? Should be changed in the interface accordingly.

Reply:
Mind elaborate the :-1:? In any case, imperative form ("Set" instead of "Sets") should be used here and in other places.